snowpack/src/types/snowpack.ts

```diff
@@ -15,11 +15,11 @@ export type SnowpackBuiltFile = {code: string | Buffer; map?: string};
 export type SnowpackBuildMap = Record<string, SnowpackBuiltFile>;
 
 /** Standard file interface */
-export interface SnowpackSourceFile {
+export interface SnowpackSourceFile<Type = string | Buffer> {
```

Comment:
We need to start a discussion on whether or not we load binary files in Snowpack (sure, we need to copy them over, but should we _read_ them?) This isn’t a quick change. 

I know this PR seems like it goes in the direction of reading buffer contents. But in actuality, we were already doing it (that’s where the bug was); this just makes it transparent.